scripts/prepare_sector_network.py

```diff
@@ -946,6 +981,96 @@ def add_storage(n, costs):
         lifetime=costs.at['H2 pipeline', 'lifetime']
     )
 
+    if options["gas_network"]:
+
+        logger.info("Add gas network")
+
+        cols = [
+            "bus0",
+            "bus1",
+            "is_bothDirection",
+            "pipe_capacity_MW",
+            "id",
+            "length_km"
+        ]
+        gas_pipes = pd.read_csv(snakemake.input.clustered_gas_network, usecols=cols)
+
+        def make_index(x):
+            connector = " <-> " if x.is_bothDirection else " -> "
+            return "Gas pipeline " + x.bus0 + connector + x.bus1
+
+        gas_pipes.index = gas_pipes.apply(make_index, axis=1)
+
+        # group parallel pipes together
+        strategies = {
+            'bus0': 'first',
+            'bus1': 'first',
+            'is_bothDirection': 'first',
+            "pipe_capacity_MW": 'sum',
+            "length_km": 'sum',
```

Comment:
Shouldn't we take the average here?
```suggestion
            "length_km": 'mean',
```